components/profile.js

```diff
@@ -65,57 +67,413 @@ SteamCommunity.prototype.editProfile = function(settings, callback) {
+											defaultval = [settings[i][t]["values"][n]["appid"], settings[i][t]["values"][n]["publishedfileid"]];
+										}
+										out.push({
+											"name": "rgShowcaseConfig[16][" + n + "][appid]",
+											"value": defaultval[0]
+										});
+										out.push({
+											"name": "rgShowcaseConfig[16][" + n + "][publishedfileid]",
+											"value": defaultval[1]
+										});
+									}
+									break;
+
+								case 'ownguides':
+									out.push({
+										"name": "profile_showcase[]",
+										"value": "12"
+									});
+
+									for (var n = 0; n < 5; n++) {
+										var defaultval = ["", ""];
+										if (settings[i][t]["values"][n] != undefined) {
+											defaultval = [settings[i][t]["values"][n]["appid"], settings[i][t]["values"][n]["publishedfileid"]];
+										}
+										out.push({
+											"name": "rgShowcaseConfig[12][" + n + "][appid]",
+											"value": defaultval[0]
+										});
+										out.push({
+											"name": "rgShowcaseConfig[12][" + n + "][publishedfileid]",
+											"value": defaultval[1]
+										});
+									}
+									break;
+							}
+
+							if (showcaseconfig[0]) {
+								for (var n = 0; n < showcaseconfig[1]; n++) {
+									var requestdata;
+									if (showcaseconfig[3][n] == undefined) {
+										requestdata = {
+											appid: 0,
+											item_contextid: 0,
+											item_assetid: 0,
+											customization_type: showcaseconfig[2],
+											slot: n,
+											sessionid: formd[0].value
+										};
+									} else {
+										requestdata = {
+											appid: showcaseconfig[3][n]["appid"],
+											item_contextid: showcaseconfig[3][n]["item_contextid"],
+											item_assetid: showcaseconfig[3][n]["item_assetid"],
+											customization_type: showcaseconfig[2],
+											slot: n,
+											sessionid: formd[0].value
+										};
+
+									}
+
+									self._myProfile("ajaxsetshowcaseconfig", requestdata, function (err, response, body) {
+										if (settings.customURL) {
+											delete selfe._profileURL;
+										}
+
+										if (err || response.statusCode != 200) {
+											if (callback) {
+												callback(err || new Error("HTTP error " + response.statusCode));
+											}
+
+											return;
+										}
+
+										// Check for an error
+										var $ = Cheerio.load(body);
+										var error = $('#errorText .formRowFields');
+										if (error) {
+											error = error.text().trim();
+											if (error) {
+												if (callback) {
+													callback(new Error(error));
+												}
+
+												return;
+											}
+										}
+
+										if (callback) {
+											callback(null);
+										}
+									});
+								}
+
+							}
+						}
+					break;
+
 			}
 		}
 
-		self._myProfile("edit", values, function(err, response, body) {
+		self._myProfile("edit", Param(out), function(err, response, body) {
```

Comment:
From my understanding of `jquery-param`, it will do what previous syntax achieved. Isn't taht right?

Reply:
I tried multiple ways to do it before, and it didn't work for me, if it actually works without Params feel free to remove it.